### DISCUSSION TITLE
create sound file "Language Code.mpq" with psx-tools from PS1 version

### DISCUSSION THREAD

**Discussion by JohnnyR8** [diasurgical/DevilutionX#7442] [Q&A] [answered]
guys i need your help please: "make all" leads to error

sudo apt-get install smpq
git clone https://github.com/diasurgical/psx-tools
cd psx-tools
# copy content of PS1 Cd into psx-tools/ps1_assets
make all
cc -o dbank.bin dbank/dbank.c
make: cc: No such file or directory
make: *** [Makefile:5: dbank.bin] Error 127

**Comment by AJenbo** [comment]
you need a C compiler installed in order to build the tool

**Reply by JohnnyR8** [reply]
thank you Ajenbo,

sudo apt install build-essential
make all
cc -o dbank.bin dbank/dbank.c
cc -o dstream.bin dstream/dstream.c
cc -o vag2wav.bin vag2wav/vag2wav.c
cc -o textdump.bin textdump/textdump.c

and how do i get now the stream3.mpg? 
tried running the sh batchfile with: sudo ./build_mpq.sh but got errors
am i missing sth?

**Reply by AJenbo** [reply]
There is no such thing as "stream3.mpg" afaik and I don't know what "sth" is either, do you have the full error message? Did you provide the PS1 data?

**Reply by JohnnyR8** [reply]
thank you for your help :-)

i thought by running  "sudo ./build_mpq.sh" the language specific streamX.mpg is generated.
copied everything from the PS1 disc into the "ps1_assets" folder and get: 
can't open file /home/ubuntu/psx-tools/ps1_assets/*.dir
./build_mpq.sh: line 33: /home/ubuntu/psx-tools/str.map: No such file or directory
smpq: /home/ubuntu/psx-tools/str.mpq: Error: Cannot create archive `/home/ubuntu/psx-tools/str.mpq': File exists
find: ‘*’: No such file or directory

am i missing something?

**Reply by AJenbo** [reply]
You mean MPQ, it's not called MPG. Also the stream files are what is on the PS1 CD, what you want to generate is a <lang_code>.mpq, so en.mpq, fr.mpq, gr.mpq, sv.mpq or jp.mpq

**Reply by AJenbo** [reply]
> copied everything from the PS1 disc into the "ps1_assets" folder and get

Maybe you can let me know what files you have in that folder then.

**Comment by JohnnyR8** [comment]
/home/ubuntu/psx-tools/ps1_assets/BLIZZARD.MOV
/home/ubuntu/psx-tools/ps1_assets/BOOTLOGO.DAT
/home/ubuntu/psx-tools/ps1_assets/BOOTLOGO.HDR
/home/ubuntu/psx-tools/ps1_assets/DIABEND1.MOV
/home/ubuntu/psx-tools/ps1_assets/DIABEND2.MOV
/home/ubuntu/psx-tools/ps1_assets/DIABEND3.MOV
/home/ubuntu/psx-tools/ps1_assets/DIABLO1.MOV
/home/ubuntu/psx-tools/ps1_assets/DIABPSX.BIN
/home/ubuntu/psx-tools/ps1_assets/DIABVIC1.MOV
/home/ubuntu/psx-tools/ps1_assets/DIABVIC2.MOV
/home/ubuntu/psx-tools/ps1_assets/DIABVIC3.MOV
/home/ubuntu/psx-tools/ps1_assets/EACLOGO.MOV
/home/ubuntu/psx-tools/ps1_assets/FPRST3.MOV
/home/ubuntu/psx-tools/ps1_assets/LEGAL.GFX
/home/ubuntu/psx-tools/ps1_assets/LICENSEA.DAT
/home/ubuntu/psx-tools/ps1_assets/LOADING.GFX
/home/ubuntu/psx-tools/ps1_assets/LUMP.BIN
/home/ubuntu/psx-tools/ps1_assets/LUMP.DIR
/home/ubuntu/psx-tools/ps1_assets/PIRACY.GFX
/home/ubuntu/psx-tools/ps1_assets/PUT_CONTENT_OF_PSX_CDROM_HERE
/home/ubuntu/psx-tools/ps1_assets/SLUS_006.19
/home/ubuntu/psx-tools/ps1_assets/STREAM1.BIN
/home/ubuntu/psx-tools/ps1_assets/STREAM1.DIR
/home/ubuntu/psx-tools/ps1_assets/STREAM2.BIN
/home/ubuntu/psx-tools/ps1_assets/STREAM2.DIR
/home/ubuntu/psx-tools/ps1_assets/STREAM3.BIN
/home/ubuntu/psx-tools/ps1_assets/STREAM3.DIR
/home/ubuntu/psx-tools/ps1_assets/STREAM4.BIN
/home/ubuntu/psx-tools/ps1_assets/STREAM4.DIR
/home/ubuntu/psx-tools/ps1_assets/SYSTEM.CNF
/home/ubuntu/psx-tools/ps1_assets/ZZZZZZZZ.ZZZ

**Reply by AJenbo** [reply, marked as answer]
what if you change the file names to lower case?

**Comment by JohnnyR8** [comment]
i will try tomorrow

a complete other question:
building Linux devX 1.6.0 works perfect, but when i try to compile the windows 32 and 64 bit versions i get this error at 87 % and abort:

[ 87%] Building CXX object Source/CMakeFiles/libdevilutionx.dir/engine/trn.cpp.obj
[ 87%] Building CXX object Source/CMakeFiles/libdevilutionx.dir/engine/render/automap_render.cpp.obj
gmake[2]: *** [Source/CMakeFiles/libdevilutionx.dir/build.make:1307: Source/CMakeFiles/libdevilutionx.dir/dvlnet/abstract_net.cpp.obj] Error 1
gmake[2]: *** Waiting for unfinished jobs....
gmake[1]: *** [CMakeFiles/Makefile2:3063: Source/CMakeFiles/libdevilutionx.dir/all] Error 2
gmake: *** [Makefile:166: all] Error 2

**Reply by AJenbo** [reply]
I don't see the error, just that there was on

**Comment by JohnnyR8** [comment]
lower case was the solution :-)

can you reproduce with ubuntu the compiling abort for the windows versions?

can i deliver more info?

**Reply by AJenbo** [reply]
Yes, but you already did in another discussion .
